File: 프로그래머스/0/181836. 그림 확대/그림 확대.js
function solution(picture, k) {
    let answer = [];
    for (let i = 0; i < picture.length; i++) {
        let stretch = []
        for (let j = 0; j < picture[i].length; j++) {
            stretch.push(picture[i][j].repeat(k))
        }
        for (let j = 0; j < k; j++) {
         answer.push(stretch.join(''))   
        }
    }
    return answer;
}

File: 프로그래머스/0/181836. 그림 확대/그림 확대.py
def solution(picture, k):
    answer = []
    for i in range(len(picture)):
        stretched_horizontal_parts = []
        for j in range(len(picture[i])):
            stretched_horizontal_parts.append(picture[i][j] * k)
        
        full_stretched_row = "".join(stretched_horizontal_parts)
        
        for _ in range(k):
            answer.append(full_stretched_row)
            
    return answer